src/mito2/src/memtable.rs

```diff
@@ -64,11 +64,19 @@ impl Default for MemtableConfig {
 pub struct MemtableStats {
     /// The estimated bytes allocated by this memtable from heap.
     estimated_bytes: usize,
-    /// The time range that this memtable contains.
+    /// The time range that this memtable contains. It is not None if
+    /// the memtable is not empty.
```

Comment:
I misread this 2 times

```suggestion
    /// The time range that this memtable contains. It is None if
    /// the memtable is empty.
```